matrix/DiagonalDifference

```diff
@@ -0,0 +1,22 @@
+#!/bin/python3
+#
+# Complete the 'diagonalDifference' function below.
+#
+# The function is expected to return an INTEGER.
+# The function accepts 2D_INTEGER_ARRAY arr as parameter.
+#
+
+def diagonalDifference(arr):
```

Comment:
You should typehint `arr` to `list[list[int]]` so that a user understands the type of `arr`.